src/main/java/de/dennisguse/opentracks/Startup.java

```diff
@@ -45,6 +45,10 @@ public void onCreate() {
         if (PreferencesUtils.shouldUseDynamicColors()) {
             DynamicColors.applyToActivitiesIfAvailable(this);
         }
+
+        if (PreferencesUtils.shouldApplyOledTheme()) {
```

Comment:
TBH: I would not use a global variable for this.
Why not check this on every `AbstractActivity.onCreate() {... PreferencesUtils.shouldApplyOledTheme() .. }`?